lunamark/reader/markdown.lua

```diff
@@ -989,14 +1096,27 @@ function M.new(writer, options)
   -- parse a link or image (direct or indirect)
   larsers.Link          = larsers.DirectLink + larsers.IndirectLink
 
-  larsers.DirectImage   = parsers.exclamation
-                        * (parsers.tag / parse_inlines)
-                        * parsers.spnl
-                        * parsers.lparent
-                        * (parsers.url + Cc(""))  -- link can be empty [foo]()
-                        * parsers.optionaltitle
-                        * parsers.rparent
-                        / writer.image
+  if options.pandoc_extensions then
+    -- Support additional attributes
```

Comment:
> Support additional attributes

Consider supporting other forms than just direct images, such as indirect images and direct/indirect links, see [Pandoc's documentation][1]. Hm, I wonder if Pandoc's `link_attributes` also supports attributes on autolinks, i.e. `<https://foo.bar/>{.class key=val}`?

 [1]: https://pandoc.org/MANUAL.html#extension-link_attributes

Reply:
Not done yet. Maybe split as a new issue for later? For now I just documented it (in the readers comments and the lunamark(1) help)